src/tracking/events.js

```diff
@@ -67,3 +80,13 @@ export const PrintMaxExceededClick = {
   },
   source: 'print_selected_max_reached_button',
 };
+
+/**
+ * NewSearchLinkClick - Analytics data dispatched on clicking "Start Over" or "Try a new search" link on results page 
+ * (data and source added before event trigger)
+ * @returns {BaseEvent}
+ */
+export const NewSearchLinkClick = {
```

Comment:
Need a 'source' here...

Reply:
I'll try to make the comment on Ln 86 clearer, source will be an empty string and data an empty object which are later set just before the event is triggered in `ResultsPage` Ln 152.